bin/appbundle-updater

```diff
@@ -129,18 +150,18 @@ App = Struct.new(:name, :repo, :bundle_without, :install_commands, :gems) do
   end
 end
 
-chef_install_command = "#{bin_dir.join("rake")} install:local"
+chef_install_command = "#{vendor_bin_dir.join("rake")} install:local"
 
-CHEFDK_APPS = [
+HABITAT_PACKAGES = [
   App.new(
```

Comment:
I believe we're using `chef/berkshelf` for the remnant of this usage.

Reply:
updated the origin to `chef` in https://github.com/chef/appbundle-updater/pull/107/commits/6ee4d2613439292b58c64673a96a6302d90eac28. Updated the origin for other packages as well.